### ISSUE TITLE
Trackables "All" option seems to not work

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#842]
**[Original report](https://bitbucket.org/zedsaidstudio/cachly/issues/842) by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

So for tonight's cache I logged and saved each TB separately when logging my cache.  What was interesting is they all logged in and the page refreshed to the correct page.   Tomorrow I'll try and just log the cache without logging any TBs.

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Correct. 

I log my find 
Select all visited
Then click into each TB and select save
When I send it, it works.  And returns me to the screen that says its logged. 

If I don't do that step then it doesn't. And doesn't save all my TB's.

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

1. I logged the find.
2. I gave it a Favorite point.
3. I VISITED ALL 8 trackables in my inventory.
4. I hit send now with 2 bars of LTE on the AT&T network.
I was at my computer when I logged it with Cachly so I could check that the log went through to my account.
So I cancelled the Found log in Cachly so I would not duplicate it.

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

  I found a cache today and after typing a log, I went into my trackables and tapped "all visited" for them (after tapping one as visited first). I then removed the one trackable that wasn't with me (my car). 

  I then saved the logs because I needed to check a couple of things on my trackables... Namely which one I just dropped off. After checking the info under the trackables menu, I then marked one as dropped off and also attached a photo to another trackable from my iPhone (first tune for that... Got the pop up about access to photos)

 I then hit send. The cache lug seemed to go okay, and then it started creating the trackable logs. It spin for a bit, and then it returned to the log, having not deleted it nor telling me that everything was sent. So, I hit "send" again. That time, it sent the cache log (again?) and then just spun while creating the trackable logs. After letting it sit for a bit, I killed the app.

I'm heading back to my desk now to see what occurred, but there may be something wrong. I did have LTE cone rag the entire time.

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Make Priority

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Yes, Cachly does choke when I use the ‘visit’ tag.  I was able to view the details of this bug on Cachly.  Interesting to note that there’s no description, but there is a goal (in German, of course).  The Google Translate does work, though.   The tracking code is V0XNGK.  I know you have integrity and will not take advantage of having a code like that.  Hope that it helps!  Also interesting to note that Mom and Max (B8814G) also doesn’t have a description as well.  For all we know, it could be that one that’s causing the issue in Cachly, and not T5-Hilfe.  Wombat Pack and A Sobbing Owl Must are my own personal trackable that are always with me, and if you think those codes would help as well, let me know.

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

**  I just had all my trackables go through without a hitch. Did you fix the bug? I don't recall seeing it in the list. If not, what I did differently this time is that I included a message and a photo on both of my problem children (T5-Hilfe and Mom & Max).** 

This could be key. We need to make sure that trackables at least have a blank string for message.
